packages/frontend/src/features/sqlRunner/components/ContentPanel.tsx

```diff
@@ -93,15 +93,17 @@ export const ContentPanel: FC<Props> = ({
         isLoading,
     } = useSqlQueryRun({
         onSuccess: (data) => {
-            if (data) {
-                dispatch(setInitialResultsAndSeries(data));
+            if (data?.results) {
+                dispatch(setInitialResultsAndSeries(data.results));
```

Comment:
In my next PR I want to pass the whole data or dispatch 2 actions
one for setting initial results and series
another for setting the xLayoutOptions and yLayoutOptions (which are defined here: https://github.com/lightdash/lightdash/pull/10801/files) 
I believe this should be separate with the new slice you're creating @magnew

Reply:
Makes sense. Although I think we figured out this afternoon that you're gonna want to add extraReducers instead of two dispatches. But you knew that :D

Reply:
100% - let's go with `extraReducers` 💪